src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -910,16 +910,19 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                 // initializers that have been analyzed but not yet lowered.
                 BoundStatementList analyzedInitializers = null;
-
+                (SyntaxNode Syntax, BoundNode Body, ExecutableCodeBinder Binder) forSemanticModel = default;
```

Comment:
>forSemanticModel [](start = 81, length = 16)

As currently written, this will cause the creation of a closure to contain this field...

Reply:
> As currently written, this will cause the creation of a closure to contain this field...

I guess we had preexisting condition with ```body```. I'll address both.

---
In reply to: [170042151](https://github.com/dotnet/roslyn/pull/24918#discussion_r170042151) [](ancestors = 170042151)